### PR TITLE
Container base image

### PR DESCRIPTION
**Pull request by joostvanzwieten**

### CODE REVIEW

File: devtools/container/build.py [hunk 1]
```diff
@@ -5,28 +5,28 @@
...
 
-if not args.build_from_worktree and not args.revision:
-    raise SystemExit('either `--revision` or `--build-from-worktree` must be specified')
+if not args.wheel and not args.revision:
+    raise SystemExit('either `--revision` or `--wheel` must be specified')
+if not args.examples and not args.revision:
+    raise SystemExit('either `--examples` or `--wheel` must be specified')
 
-rev = args.revision or 'HEAD'
+rev = args.revision
```
Comment (gertjanvanzwieten) [inline devtools/container/build.py:26]: removing HEAD here seems incompatible with leaving revision unspecified above.

File: devtools/container/build_base.py [hunk 2]
```diff
@@ -19,8 +19,7 @@
     container.run('apt', 'update')
     # Package `libtbb2` is required when using Intel MKL with environment
     # variable `MKL_THREADING_LAYER` set to `TBB`, which is nowadays the default.
-    container.run('apt', 'install', '-y', '--no-install-recommends', 'python3', 'python3-pip', 'python3-wheel', 'python3-ipython', 'python3-numpy', 'python3-scipy', 'python3-matplotlib', 'python3-pil', 'python3-meshio', 'libmkl-rt', 'libtbb2', env=dict(DEBIAN_FRONTEND='noninteractive'))
-    container.run('pip', 'install', '--no-cache-dir', 'bottombar', 'treelog', 'stringly')
+    container.run('apt', 'install', '-y', '--no-install-recommends', 'python3', 'python3-pip', 'python3-wheel', 'python3-ipython', 'python3-numpy', 'python3-scipy', 'python3-matplotlib', 'python3-pil', 'python3-meshio', 'libmkl-rt', 'libomp-dev', 'libtbb2', 'python3-gmsh', env=dict(DEBIAN_FRONTEND='noninteractive'))
```
Comment (gertjanvanzwieten) [inline devtools/container/build_base.py:22]: let's move meshio to the nutils image

**Review by gertjanvanzwieten — CHANGES_REQUESTED** [review]

File: devtools/container/build.py (hunk 1 above)
Comment (joostvanzwieten) [reply devtools/container/build.py]: fixed

File: devtools/container/build_base.py (hunk 2 above)
Comment (joostvanzwieten) [reply devtools/container/build_base.py]: done

**Review by gertjanvanzwieten — APPROVED** [review]

**Pull request merged**